src/components/Auth/SignUp.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import Input from "@material-ui/core/Input";
+import InputLabel from "@material-ui/core/InputLabel";
+import FormControl from "@material-ui/core/FormControl";
+import Button from "@material-ui/core/Button";
+import { withStyles } from "@material-ui/styles";
+
+const styles = {
+  button: {
+    marginTop: 30
+  }
+};
+
+class SignUp extends Component {
```

Comment:
Use PropTypes with form, email, name and etc with types

Reply:
I fixed it only for classes. I do not have other properties in this component.